tecken/tests/test_sentry.py

```diff
@@ -234,19 +234,16 @@ def test_count_sentry_scrub_error(metricsmock):
 
 
 def test_sentry_wrap_non_app_error_has_release():
-    port = os.environ.get("EXPOSE_SENTRY_PORT", 8090)
-    fakesentry_url = f"http://fakesentry:{port}/"
+    fakesentry_url = f"http://fakesentry:8090/"
```

Comment:
this shouldn't have been checking `EXPOSE_SENTRY_PORT`, because that's for use with `localhost` when outside of a docker container. when inside a container the port is always 8090.